test/mlir/onnx/onnx_constprop_no_shape_inference.mlir

```diff
@@ -64,3 +64,36 @@ func @test_splitv11_axis_1_no_splitattr() -> (tensor<2x5xf32>, tensor<2x5xf32>)
   // CHECK: {{.*}}  = "onnx.Constant"() {value = dense<{{\[}}[5.000000e+00, 6.000000e+00, 7.000000e+00, 8.000000e+00, 9.000000e+00], [1.500000e+01, 1.600000e+01, 1.700000e+01, 1.800000e+01, 1.900000e+01]]> : tensor<2x5xf32>} : () -> tensor<2x5xf32>
   // CHECK-NOT: {{.*}} = "onnx.SplitV11"{{.*}}
 }
+
```

Comment:
@DavidChan0519 could you please add `// -----` in between these two tests to see if it helps make Jenkins ppc64le passes? Thanks!

Reply:
After added // -----, I ran the cmd below repeatly :
cmake --build . --target check-onnx-lit
All pass.